Makefile

```
@@ -2,13 +2,23 @@
 # https://github.com/precice/calculix-adapter/wiki/Installation-instructions-for-CalculiX
 # Set the following variables before building:
 # Path to original CalculiX source (e.g. $(HOME)/ccx_2.16/src )
-CCX             = $(HOME)/PathTo/CalculiX/ccx_2.16/src
-# Path to SPOOLES main directory (e.g. $(HOME)/SPOOLES.2.2 )
-SPOOLES         = $(HOME)/PathTo/SPOOLES
-# Path to ARPACK main directory (e.g. $(HOME)/ARPACK )
-ARPACK          = $(HOME)/PathTo/ARPACK
-# Path to yaml-cpp prefix (e.g. $(HOME)/yaml-cpp, should contain "include" and "build")
-YAML            = $(HOME)/PathTo/yaml-cpp
+CCX             = $(HOME)/CalculiX/ccx_2.16/src
```

We could deduce this from the location of the Makefile as described [here](https://stackoverflow.com/a/18137056/11633170). That would be one less thing to worry about

_untested suggestion:_
```suggestion
CCX             = $(strip $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
```

---

If I understand correctly, this will give the current directory (i.e. `calculix-adapter`). What we want is the path to the vanilla CalculiX directory, which can be anywhere in the system.

---

Right, sorry! 

Maybe it would be simpler to specify where to extract the sources instead. There are only a few MB and would remove another customization point in the installation procedure.
We could even provide a `download_ccx.sh` for that purpose.

---

Good point, I made this clearer in the [documentation](https://github.com/precice/calculix-adapter/wiki/Installation-instructions-for-CalculiX#get-the-source).

However, I am not sure if providing a `download_ccx.sh` script would be a good idea: it would probably confuse some users (especially those that have CalculiX already), or at least more than the current variant confuses most users.